src/pykiso/test_coordinator/test_case.py

```diff
@@ -183,6 +172,61 @@ def setUpClass(cls) -> None:
         if options.report_type == "junit":
             initialize_logging(None, options.log_level, options.report_type)
 
+    def setUp(self) -> None:
```

Comment:
Do we need to define them?

Reply:
It's still good to have them for autocompletion and implementing them doesn't change anything on user side (these are already the default names defined by unittest) so I'm fine with it